app/app/src/main/jni/vsp-pty.cpp

```diff
@@ -195,79 +190,53 @@ static void* ptyThread(void* irrelevant)
         {
             //            buf[nread] = '\0';
 
-            if (CHECK_BIT(*buf, TIOCPKT_DATA))
```

Comment:
Here *buf must be compared directly with the defined constants, they are not bit masks